python_modules/dagster/dagster/__init__.py

```diff
@@ -244,11 +246,54 @@
     execute_solid_within_pipeline,
     execute_solids_within_pipeline,
 )
+from pep562 import pep562
 
 from .version import __version__
 
 from dagster.config.source import BoolSource, StringSource, IntSource  # isort:skip
 
+_DEPRECATED = {
+    "EventMetadataEntry": ("MetadataEntry", MetadataEntry),
+    "EventMetadata": ("MetadataValue", MetadataValue),
+    "TextMetadataEntryData": ("TextMetadataValue", TextMetadataValue),
+    "UrlMetadataEntryData": ("UrlMetadataValue", UrlMetadataValue),
+    "PathMetadataEntryData": ("PathMetadataValue", PathMetadataValue),
+    "JsonMetadataEntryData": ("JsonMetadataValue", JsonMetadataValue),
+    "MarkdownMetadataEntryData": ("MarkdownMetadataValue", MarkdownMetadataValue),
+    "PythonArtifactMetadataEntryData": ("PythonArtifactMetadataValue", PythonArtifactMetadataValue),
+    "FloatMetadataEntryData": ("FloatMetadataValue", FloatMetadataValue),
+    "IntMetadataEntryData": ("IntMetadataValue", IntMetadataValue),
+    "DagsterPipelineRunMetadataEntryData": (
+        "DagsterPipelineRunMetadataValue",
+        DagsterPipelineRunMetadataValue,
+    ),
+    "DagsterAssetMetadataEntryData": ("DagsterAssetMetadataValue", DagsterAssetMetadataValue),
+    "TableMetadataEntryData": ("TableMetadataValue", TableMetadataValue),
+    "TableSchemaMetadataEntryData": ("TableSchemaMetadataValue", TableSchemaMetadataValue),
```

Comment:
seems odd we have to manually double encode the value in the tuple, is programatic not an option?